### DISCUSSION TITLE
AllowDrop property gives error

### DISCUSSION THREAD

**Discussion by logixoul** [AvaloniaUI/Avalonia#16267] [Q&A]
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="TonemapsterUI.TestcaseWindow"
        Title="TestcaseWindow">
	<Border Name="MyBorder" AllowDrop="True"/>
</Window>

```
I get:
`Unable to resolve suitable regular or attached property AllowDrop on type Avalonia.Controls:Avalonia.Controls.Border`

Any ideas please? :)

**Comment by rabbitism** [comment]
`DragDrop.AllowDrop="True"`

**Reply by logixoul** [reply]
@rabbitism Thanks, yes, I also tried that but got the same result.

**Reply by timunie** [reply]
The exact same error message? I doubt it.

**Reply by logixoul** [reply]
@timunie 
Ok, sorry, you're right, applying @rabbitism 's change fixes the problem, but when I do this:
`<Border Name="MyBorder" DragDrop.AllowDrop="True" DragDrop.DragEnter="OnDragEnter"/>`
... I get this:
`Unable to resolve suitable regular or attached property DragEnter on type Avalonia.Base:Avalonia.Input.DragDrop`
... which is basically the same old message.

**Reply by timunie** [reply]
That's different thing. The event needs to be setup in code behind rn. Check docs, control catalog and other discussions about dnd
